src/commissaire_http/authentication/httpbasicauth.py

```diff
@@ -105,7 +105,11 @@ def check_authentication(self, user, passwd):
         valid = False
         hashed = self._data[user]['hash']
         try:
-            if bcrypt.hashpw(passwd.encode('utf-8'), hashed) == hashed:
+            encoded_passwd = passwd.encode('utf-8')
+            encoded_hashed = hashed.encode('utf-8')
```

Comment:
Yeah, that's the incompatibility I hit.  `py-bcrypt` wants strings, `bcrypt` wants bytes.

https://github.com/pyca/bcrypt/blob/master/src/bcrypt/__init__.py#L61